convert2rhel/pkghandler.py

```diff
@@ -225,9 +225,16 @@ def _enable(self):
             set_releasever=False,
         )
         if ret_code:
-            loggerinst.critical(
+            loggerinst.critical_no_exit(
                 "Failed to install subscription-manager packages. See the above yum output for details."
             )
+            raise exceptions.CriticalError(
+                id_="FAILED_TO_INSTALL_SUBSCRIPTION_MANAGER_PACKAGES",
+                title="Failed to install subscription-manager packages.",
+                description="To be able to subscribe the system to Red Hat we need subscription-manager packages to do so. Without these packages we cannot subscribe the system and we cannot install Red Hat Enterprise Linux packages.",
+                diagnosis="Failed to install packages %s. Errno: %s, Error: %s"
```

Comment:
Maybe we should use `status` or `return code` instead of `errno`?

Reply:
I think @jochapma is using `status` in one of his PRs too

Reply:
We used the same in backup.py so I just kept with the consistency

Reply:
True, though the distingtion between output and status from a command call and an exception being thrown might be good, so keeping errno and error for exceptions might be good while commands are output and status, like Joe said

Can always change later